library/network_connections.py

```diff
@@ -1553,24 +1553,35 @@ def _complete_kwargs_loglines(self, rr, connections, idx):
                 c["persistent_state"],
             )
             prefix = prefix + (", '%s'" % (c["name"]))
-        for r in rr["log"]:
-            yield (r[2], "[%03d] %s %s: %s" % (r[2], LogLevel.fmt(r[0]), prefix, r[1]))
+        for severity, msg, idx in rr["log"]:
+            yield (
+                idx,
+                "[%03d] %s %s: %s" % (idx, LogLevel.fmt(severity), prefix, msg),
+                severity,
+            )
 
-    def _complete_kwargs(self, connections, kwargs, traceback_msg=None):
+    def _complete_kwargs(self, connections, kwargs, traceback_msg=None, fail=False):
         if "warnings" in kwargs:
-            logs = list(kwargs["warnings"])
+            warning_logs = list(kwargs["warnings"])
         else:
-            logs = []
+            warning_logs = []
 
+        debug_logs = []
         loglines = []
         for res in self._run_results:
             for idx, rr in enumerate(res):
                 loglines.extend(self._complete_kwargs_loglines(rr, connections, idx))
         loglines.sort(key=lambda x: x[0])
-        logs.extend([x[1] for x in loglines])
+        for x in loglines:
```

Comment:
Please use names for `x[1]` etc here, too

Reply:
Fixed, thanks!